qt/python/mantidqt/widgets/memorywidget/memoryview.py

```diff
@@ -6,9 +6,9 @@
 # SPDX - License - Identifier: GPL - 3.0 +
 #  This file is part of the mantid workbench.
 #
-#
 from qtpy.QtWidgets import QWidget, QProgressBar
 from qtpy.QtCore import Qt
+from PyQt5.QtCore import pyqtSlot, pyqtSignal
```

Comment:
Merge with line above so we can use `qtpy`
```
from qtpy.QtCore import Qt, pyqtSlot, pyqtSignal
```

Reply:
I would do that, but it seems that at runtime it doesn't find pyqtSlot or pyqtSignal

Reply:
Just found that the replacement is Slot and Signal I will make the changes and push it back up!